ath.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+cd $(dirname $0)
+
+# https://github.com/jenkinsci/acceptance-test-harness/releases
+ATH_VERSION=5458.v911b_2f0818ee
+
+# TODO use Artifactory proxy?
```

Comment:
I'd like to fix the Azure provider first (cf https://github.com/jenkins-infra/helpdesk/issues/3221)

Reply:
I'll take a look, thanks for the suggestion.